simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/configuration/SimulatorConfiguration.java

```diff
@@ -116,13 +116,13 @@
          * List of events that defines when a new transaction should start.
          */
         @Builder.Default
-        private List<String> txStartPoint = DEFAULT_TX_START_POINT;
+        private List<String> txStartPoint = DEFAULT_TX_START_POINTS;
```

Comment:
how about txStartPoint**s** and txStopPoint**s** ? :)

Reply:
Done